homeassistant/components/plugwise/sensor.py

```diff
@@ -219,6 +220,69 @@
         device_class=SensorDeviceClass.ENERGY,
         state_class=SensorStateClass.TOTAL_INCREASING,
     ),
+    SensorEntityDescription(
+        key="electricity_phase_one_consumed",
+        name="Electricity phase one consumed",
+        device_class=SensorDeviceClass.POWER,
+        native_unit_of_measurement=UnitOfPower.WATT,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    SensorEntityDescription(
+        key="electricity_phase_two_consumed",
+        name="Electricity phase two consumed",
+        device_class=SensorDeviceClass.POWER,
+        native_unit_of_measurement=UnitOfPower.WATT,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    SensorEntityDescription(
+        key="electricity_phase_three_consumed",
+        name="Electricity phase three consumed",
+        device_class=SensorDeviceClass.POWER,
+        native_unit_of_measurement=UnitOfPower.WATT,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    SensorEntityDescription(
+        key="electricity_phase_one_produced",
+        name="Electricity phase one produced",
+        device_class=SensorDeviceClass.POWER,
+        native_unit_of_measurement=UnitOfPower.WATT,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    SensorEntityDescription(
+        key="electricity_phase_two_produced",
+        name="Electricity phase two produced",
+        device_class=SensorDeviceClass.POWER,
+        native_unit_of_measurement=UnitOfPower.WATT,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    SensorEntityDescription(
+        key="electricity_phase_three_produced",
+        name="Electricity phase three produced",
+        device_class=SensorDeviceClass.POWER,
+        native_unit_of_measurement=UnitOfPower.WATT,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    SensorEntityDescription(
+        key="voltage_phase_one",
+        name="Voltage phase one",
+        device_class=SensorDeviceClass.VOLTAGE,
+        native_unit_of_measurement=UnitOfElectricPotential.VOLT,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    SensorEntityDescription(
+        key="voltage_phase_two",
+        name="Voltage phase two",
+        device_class=SensorDeviceClass.VOLTAGE,
+        native_unit_of_measurement=UnitOfElectricPotential.VOLT,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    SensorEntityDescription(
+        key="voltage_phase_three",
+        name="Voltage phase three",
+        device_class=SensorDeviceClass.VOLTAGE,
+        native_unit_of_measurement=UnitOfElectricPotential.VOLT,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
```

Comment:
Please address this comment in a new PR.

Voltage sensors should be disabled by default, as per developer documentation:

<img width="813" alt="CleanShot 2023-01-08 at 17 18 32@2x" src="https://user-images.githubusercontent.com/195327/211207720-f83a53f3-a6d5-4628-9e6a-ad2162794cc7.png">

Ref: https://developers.home-assistant.io/docs/core/entity

/CC: @elupus

Reply:
This is not battery voltage?

Reply:
Frenck is right, this is line voltage of the invididual line phase leads

Reply:
i.e. The P1 is used for DSMR monitoring of gas / electricity (not to confuse with the climate parts)

Reply:
> This is not battery voltage?
> 
> Ps. But I can sort of accept that it should be disabled by default since its rather not much use to know mains voltage.

Yeah, the example is a battery, but the general consensus in that example applies. Voltage is really rare to use, but it fluctuates a lot (thus causing lots of data recording, mostly unneeded).